### PR TITLE
Additional stop instructions, additional sass

### PR DESCRIPTION
**Pull request by blagh**

### CODE REVIEW

File: bot.js [hunk 1]
```diff
@@ -143,6 +143,7 @@ controller.hears(["o/", "O/", "0/", "\\\\o", "\\\\O", "\\\\0"], ["ambient"], fun
 				'Too slow!',
 				`Hey look, everyone!  ${response.user.profile.real_name} can't count!`,
 				'Awww, maybe next time.',
+				"It's OK, I don't get any coffee either",
```
Comment (konrad0101) [inline bot.js]: Needs a "." at the end of the sentence.  =)

Comment (DanHulton) [inline bot.js]: Missing "." at the end of the sentence. =)

Comment (blagh) [reply bot.js]: Engh.

Comment (blagh) [reply bot.js]: But I'm lazy

**Pull request merged**